crates/ruma-client-api/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Improvements:
 
 * Add `From<&UserId>` and `From<&OwnedUserId>` implementations for `UserIdentifier`
 * Add `UserIdentifier::email` constructor
+* Add unstable support for querying relating events (MSC2675)
```

Comment:
Wrong section, this should go into unreleased (might be the cause of an auto-merge that this is wrong). There should also be an entry about `RelationType` moving from ruma-client-api` to `ruma-common`.